<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>random</class>
 <widget class="QDialog" name="random">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>news</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pictures.qrc">
    <normaloff>:/pictures/logo.png</normaloff>:/pictures/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>                           random</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MyPushButton" name="choice1">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MyPushButton" name="choice2">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(182,182,182);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MyPushButton" name="choice3">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(182,182,182);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
